pkg/adapters/hertz/.gitignore

```diff
@@ -0,0 +1,66 @@
+# JetBrains template
```

Comment:
gitignore is not required, it is already available in the main project

Reply:
GET